app/models/project.rb

```diff
@@ -13,6 +13,7 @@
 # Indexes
 #
 #  index_projects_on_deleted_at    (deleted_at)
+#  index_projects_on_name          (name)
```

Comment:
why? it may be a pretty big index..

Reply:
because i search by name

Reply:
index is not for search, it's for lookup
it's useful for queries like `find_by(name: '') #=> exact match` not `LIKE or ILIKE` partial match